src/promptflow/tests/_constants.py

```diff
@@ -13,3 +13,4 @@
 DEFAULT_WORKSPACE_NAME = "promptflow-eastus"
 DEFAULT_RUNTIME_NAME = "test-runtime-ci"
 DEFAULT_REGISTRY_NAME = "promptflow-preview"
+CI_CLI_USER_AGENT = "perf_monitor/1.0"
```

Comment:
Nit: CLI_PERF_MONITOR_AGENT?

Reply:
Sure, I can make the next PR modification.